templates/ez_segmenter.ipynb

```diff
@@ -0,0 +1,1019 @@
+{
```

Comment:
<p>Let's remove the print statement, if there are many FOVs in <code>tiff_dir</code> this will product too much output. A user could always inspect it manually themselves later on.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/ark-analysis/pull/958/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
<p>Done</p>